cmd/oras/pull.go

```diff
@@ -148,9 +148,29 @@ func runPull(opts pullOptions) error {
 		return ret, nil
 	}
 
+	ctx, _ := opts.SetLoggerLevel()
+	var dst = file.New(opts.Output)
+	dst.AllowPathTraversalOnWrite = opts.PathTraversal
+	dst.DisableOverwrite = opts.KeepOldFiles
+
 	pulledEmpty := true
 	copyOptions.PreCopy = display.StatusPrinter("Downloading", opts.Verbose)
 	copyOptions.PostCopy = func(ctx context.Context, desc ocispec.Descriptor) error {
+		// restore successors, if applicable
+		ss, err := content.Successors(ctx, dst, desc)
+		if err != nil {
+			return err
+		}
+		for _, s := range ss {
+			exists, _ := dst.Exists(ctx, s)
```

Comment:
if `desc.Annotations[ocispec.AnnotationTitle]` is not presented, then `desc` won't be downloaded to `dst`. I think that should be used here instead of `.Exists`

Reply:
replaced the `exists` check with `_, ok := s.Annotations[ocispec.AnnotationTitle]`.